platform/javascript/audio_driver_javascript.cpp

```diff
@@ -36,6 +36,15 @@
 
 AudioDriverJavaScript *AudioDriverJavaScript::singleton = nullptr;
 
+bool AudioDriverJavaScript::is_available() {
+	return EM_ASM_INT({
```

Comment:
Doesn't this need an explicit `(bool)` cast? (If there isn't an `EM_ASM_BOOL`).

Reply:
(Unrelated, but you lost the end of the commit title :P)

Reply:
Well, there is no `EM_ASM_BOOL`, and int to bool seem safe in this case, but I appended `!= 0` at the end so the function returns a boolean.